### ISSUE TITLE
Attention-Unet implemetation 

### ISSUE THREAD

**Issue by k-nayak** [milesial/Pytorch-UNet#300]
Hello, 

I tried to implement Attention Unet in the unet_model.py and unet_parts.py to see how it well it works to traditional Unet. I have faced an issue where the training stops immediately after it starts. 
As shown below: 

INFO: Using device cuda
INFO: Network:
        3 input channels
        1 output channels (classes)
        Bilinear upscaling
INFO: Creating dataset with 180 examples
2021-09-27 08:36:35.793731: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library cudart64_110.dll
INFO: Starting training:
        Epochs:          2
        Batch size:      1
        Learning rate:   0.0001
        Training size:   162
        Validation size: 18
        Checkpoints:     True
        Device:          cuda
        Images scaling:  0.5

Epoch 1/2:   0%|                                    | 0/162 [00:00<?, ?img/s]C:\Users\NAK4SI\.conda\envs\NAK4SI\lib\site-packages\torch\nn\functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  ..\c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
Epoch 1/2:   1%|        | 1/162 [00:06<16:36,  6.19s/img, loss (batch)=0.818]global step: 1
batch_size: 1
n_train: 162
Epoch 1/2:   1%|        | 1/162 [00:06<17:11,  6.41s/img, loss (batch)=0.818]#

I am not sure what could be the issue here. I have been getting a Dice of 0.7 in U-Net and want to compare it with Attention Unet. If anyone can guide me where to look for the issue it would be very helpful. Thanks in advance. 
I can share the code if needed. :)

**Comment by Sehjbir** [comment]
were you able to run attention-unet? can you update what changes did you make?
